### DISCUSSION TITLE
Any way to make the fancy title bar background transparent?

### DISCUSSION THREAD

**Discussion by ehakan** [wezterm/wezterm#4816] [Q&A] [answered]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20230712-072601-f4abf8fd

### Ask your question!

Is there an option for adjusting the opacity of the fancy titlebar background itself?
I couldn't find such an option and currently my tab bar has a flat background, clashing with the blurry style of the window background .

Basically I want to get rid of this bar at the bottom, but retain the tabs themselves.

<img width="760" alt="Screenshot 2024-01-20 at 3 38 54 PM" src="https://github.com/wez/wezterm/assets/20271015/5e7b4d68-8ef8-4683-9d12-9774014f7486">

Relevant configuration snippet:
```lua
config.window_decorations = "RESIZE"
config.use_fancy_tab_bar = true
config.tab_bar_at_bottom = true
config.hide_tab_bar_if_only_one_tab = true
config.window_frame = {
        -- no idea how to make these transparent
	inactive_titlebar_bg = selected_colors.background,
	active_titlebar_bg = selected_colors.background,
}
config.window_background_opacity = 0.85
config.macos_window_background_blur = 50
```

I tried setting `config.window_frame.inactive_titlebar_bg` and  `config.window_frame.active_titlebar_bg` to `#00000000` hoping maybe it was compatible with alpha values but wezterm did not allow it.

**Comment by wez** [comment, marked as answer]
See:
* https://github.com/wez/wezterm/issues/3580

once c607c6a59c572706fe92dcb71aeee28315bcf59d shows up in nightly builds (typically takes 30-60 minutes, depending on the OS), you can do:

```lua
config.window_frame = {
        inactive_titlebar_bg = "none",
        active_titlebar_bg = "none",
}
```

(`none` is an alias for transparent black)

**Reply by ehakan** [reply]
Thank you, got the nightly and it worked perfectly!

<img width="854" alt="image" src="https://github.com/wez/wezterm/assets/20271015/e9c96fc5-7d2a-4746-8218-80ac95bd4a32">

(also, damn, that was fast)

**Reply by J-Siu** [reply]
On MacOS, I am using following config:
```lua
return {
	use_fancy_tab_bar = true,
	window_background_opacity = 0.85,
	window_close_confirmation = 'NeverPrompt',
	window_decorations = 'RESIZE|INTEGRATED_BUTTONS',
	window_frame = {
		-- Title Bar
		active_titlebar_bg = 'none',
		inactive_titlebar_bg = 'none',
		-- Border
		border_bottom_color = '#3b3b3b',
		border_bottom_height = '1px',
		border_left_color = '#3b3b3b',
		border_left_width = '1px',
		border_right_color = '#3b3b3b',
		border_right_width = '1px',
		border_top_height = '0px',
		-- Font
		font = wezterm.font('IosevkaTerm NFM'),
		font_size = 12,
	},
}
```

The title bar is completely transparent, while still clickable/draggable

<img width="858" height="643" alt="Screenshot 2025-09-12 at 16 23 36" src="https://github.com/user-attachments/assets/3b006309-c03f-4672-a298-fc85aef9da3c" />

Is it possible to set the transparency percentage like `window_background_opacity`?
